### DISCUSSION TITLE
QRS@NTU Research Project Overview - Sep 6, 2026

### DISCUSSION THREAD

**Discussion by yuhesui** [Quantitative-Research-Society-NTU/QRSNTU#8] [General]
<html>
<body>
<html><head></head><body><h1>QRS@NTU Research Project Overview</h1><p><strong>Snapshot date:</strong> Sep 6, 2026 </p><p>QRS@NTU research is organised into several <strong>project batches</strong>. A batch indicates the context in which a group of projects was initiated—for example, a NeurIPS research cycle, a financial-risk research programme, or a competition—not necessarily the final publication venue.</p><hr><h2>Project Overview</h2>

| Project Batch | Project Title | Rough Description | Current Stage | Submitted / Completing To | Planned Next Submission | Recruitment
-- | -- | -- | -- | -- | -- | --
NeurIPS Batch | GeoAttention — Geometry-Adaptive Attention | Studies how attention can exploit low-dimensional task structure by adapting its effective geometry. Combines approximation/statistical theory with controlled attention experiments. | Mature research; current paper submitted | NeurReps 2026 Proceedings — Submitted / Under Review | Possible 2027 full ML/theory paper if substantial additional results are obtained | Closed
NeurIPS Batch | SoftRank — Approximation Complexity of Softmax Attention | Studies the minimum rank required to approximate softmax attention and how this depends on temperature, support geometry and effective query–key interaction dimension. | Mature theory programme; full-paper development continuing | NeurReps 2026 Extended Abstract — Submitted / Under Review | Potential COLT / ICML / other 2027 theory venue | Closed
NeurIPS Batch | GeoAFT-I — Geometric Attention Fine-Tuning | Studies whether explicitly modifying query–key compatibility geometry provides capabilities beyond conventional Q/K fine-tuning. Includes low-rank theory, synthetic interventions and pretrained-model tests. | Phase I completed | NeurReps 2026 Proceedings — Submitted / Under Review | Phase I largely closed; continued through GeoAFT-II | Closed
NeurIPS Batch | GeoAFT-II — Attention Identifiability Across Pre- and Post-Training | Investigates whether apparent changes in attention geometry after post-training correspond to genuine functional changes or can arise from equivalent parameterisations. | Workshop paper submitted; larger programme active | Pre-Training → Post-Training @ NeurIPS 2026 — Submitted / Under Review | Potential 2027 full ML conference paper after broader experimental coverage | Closed
NeurIPS Batch | PolicyAttention — Transformers as Policy-Improvement Algorithms | Studies whether Transformers can represent and learn policy mirror descent and temporal-difference evaluation in context, combining RL theory with learned Transformer experiments. | Mature research; archival extension active | PTA @ NeurIPS 2026 — Submitted / Under Review | AAMAS 2027 Main Track — in development | Closed
NeurIPS Batch | AI-Native Workflow — Hierarchical AI Research Systems | Develops a hierarchical research workflow in which coordinating agents, execution agents and verifiers carry out long-horizon research while maintaining evidence, recoverability and human control. | Implemented system; several papers and artifacts developed | Who Verifies the Agents? @ NeurIPS 2026 — Submitted; NeurIPS 2026 Education — Submitted / Under Review; Meta-Agents @ NeurIPS 2026 — approximately complete; AutoMLR 2026 — final submission compilation | Longer-term agentic-AI / ML-systems paper after broader empirical evaluation | Closed
ICAIF / FinRisk Batch | Certified FRTB Hedge Screening | Studies whether low-rank or compressed regulatory-risk representations can give auditable guarantees that important hedge-selection and capital-sensitive decisions are preserved. | Early research; primary near-term workshop candidate | — | ICAIF 2026 Workshop / RAIOps4Fin — planned | Finalising
ICAIF / FinRisk Batch | Decision-Preserving Rank for FX Hedging | Studies how low-rank a representation can be while still preserving an optimal or near-optimal FX-options hedge, rather than merely approximating prices or Greeks accurately. | Early mathematical research | — | Potential ICAIF 2026 workshop preliminary paper and/or later 2027 theory/quant-finance paper | Finalising
ICAIF / FinRisk Batch | Low-Rank FRTB Risk Attribution & Optimisation | Studies compressed representations of FRTB risk that preserve capital, trade-level attribution, hedge ranking and downstream optimisation decisions. | Early theory and methodology development | — | Potential ICAIF 2026 workshop, followed by fuller 2027 work | Finalising
NLPCC Batch | AlphaAllocator — Bounded Transformer News Overlays for Asset Allocation | Combines transparent portfolio allocation with bounded Transformer-based financial-news relevance and sentiment signals, using leakage-aware evaluation and explicit risk controls. | Main project completed; limited follow-on research possible | NLPCC 2026 — Accepted Poster; Track 1 Winner / Track 2 6th | Possible ICAIF-oriented follow-on if additional experiments produce a genuinely distinct result | Closed
CFFA Competition Batch | HealthFormer-RL — Risk-Sensitive Retirement Portfolio Control | Long-horizon retirement portfolio optimisation incorporating investment decisions together with health-state uncertainty, mortality, medical expenditure and feasibility constraints using recurrent/RL methods. | Competition project developed into a research paper | ACM ICAIF 2026 Main Track — Submitted / Under Review | Await ICAIF decision before deciding on a further extension | Closed
CFFA Competition Batch | Conditional Pricing of Greenwashing Risk | Studies mispricing and market discipline in China's labelled green-bond market, including whether ex-ante greenwashing risk affects issuance spreads and whether regulatory tightening changes that relationship. | Competition research project / empirical paper under development | CFFA 2026 Competition | Potential finance / sustainable-finance publication after competition-stage completion | Closed
Agenthon Competition Batch | Agenthon T2/T4 — Robust Financial Forecasting & Grounded Prediction | Competition-oriented research on robust financial prediction under sealed evaluation. The current focus is on reasoning-augmented time-series forecasting and evidence-grounded tabular prediction, with leakage-safe validation, provenance, reliability and hidden-test robustness. | Active competition development | Agenthon 2026 — Tracks T2 and T4 | Competition final submission; potentially reusable as later financial-ML research if a sufficiently interesting result emerges | Closed
SimulacraBench Batch | SimulacraBench — Probabilistic Respondent Modelling | Benchmark project on probabilistic respondent-by-question prediction: reconstructing survey-response distributions across a large international respondent × question matrix under constrained evaluation. | Competition / benchmark project | SimulacraBench — Tracks A/B | Benchmark submission and, depending on placement/results, potential associated proceedings contribution | Closed

<hr><h1>Recruitment</h1><h2>Current Research Recruitment</h2><h3><strong>ICAIF / FinRisk Workshop Batch — Finalising</strong></h3><p>This is currently the <strong>main batch open to new QRS@NTU researchers</strong>.</p><p>The batch focuses on research at the intersection of:</p><ul><li><p>quantitative finance;</p></li><li><p>machine learning and optimisation;</p></li><li><p>derivatives and FX options;</p></li><li><p>market and regulatory risk;</p></li><li><p>FRTB;</p></li><li><p>low-rank / compressed representations;</p></li><li><p>decision-preserving approximation;</p></li><li><p>hedge selection and portfolio optimisation; and</p></li><li><p>interpretable or auditable financial ML.</p></li></ul><p>Current candidate projects include:</p><ol><li><p><strong>Certified FRTB Hedge Screening</strong><br>Develop low-rank or compressed risk representations while certifying that important hedge-selection decisions are preserved.</p></li><li><p><strong>Decision-Preserving Rank for FX Hedging</strong><br>Study the minimum approximation complexity required to preserve the optimal or near-optimal hedge, rather than requiring every intermediate financial quantity to be reconstructed accurately.</p></li><li><p><strong>Low-Rank FRTB Risk Attribution &amp; Optimisation</strong><br>Investigate whether portfolio-level regulatory-risk measures can be compressed or decomposed in ways that enable trade attribution and downstream optimisation.</p></li></ol><p>The immediate objective is to develop one or more sufficiently strong preliminary results for the <strong>ICAIF 2026 workshop cycle</strong>, while stronger directions may continue as longer-term 2027 research projects.</p><p><strong>Recruitment status: OPEN</strong></p><p>New researchers may contribute to areas including:</p><ul><li><p>mathematical formulation and theorem development;</p></li><li><p>quantitative-finance modelling;</p></li><li><p>optimisation;</p></li><li><p>simulation and synthetic-data construction;</p></li><li><p>empirical ML experiments;</p></li><li><p>FRTB / regulatory-risk implementation;</p></li><li><p>literature review and novelty analysis; and</p></li><li><p>research engineering and reproducible evaluation.</p></li></ul><hr><h2>Currently Closed to New Recruitment</h2><h3><strong>NeurIPS Batch — Closed</strong></h3><p>The 2026 NeurIPS-oriented batch has reached a mature stage. Most projects have already been submitted, are under review, or are being converted into follow-on full papers.</p><p>This includes:</p><ul><li><p>GeoAttention;</p></li><li><p>SoftRank;</p></li><li><p>GeoAFT-I;</p></li><li><p>GeoAFT-II;</p></li><li><p>PolicyAttention; and</p></li><li><p>AI-Native Workflow.</p></li></ul><p>These projects therefore <strong>are not currently recruiting new researchers</strong>. Existing members may continue follow-on work, and individual projects may reopen later if a sufficiently distinct 2027 research phase is initiated.</p><h3><strong>NLPCC Batch — Closed</strong></h3><p>AlphaAllocator has completed its primary research and submission cycle and is not currently recruiting.</p><h3><strong>CFFA Competition Batch — Closed</strong></h3><p>HealthFormer-RL and the China green-bond / greenwashing-risk project are existing competition-originated projects and are not currently open as general research projects.</p><h3><strong>Agenthon Competition Batch — Closed</strong></h3><p>Agenthon T2/T4 is already in its active competition-development cycle with an established execution scope. It is listed here for visibility but is not currently a general recruitment project.</p><h3><strong>SimulacraBench Batch — Closed</strong></h3><p>SimulacraBench is similarly listed as an active benchmark/competition effort rather than an open QRS research project.</p><hr><h1>Notes and Description</h1><h2>Project batches</h2><p>A <strong>project batch</strong> describes the research context in which a project was initiated. It does not restrict where the resulting research can eventually be published.</p><p>For example:</p><ul><li><p><strong>HealthFormer-RL</strong> originated from the <strong>CFFA Competition Batch</strong> and was subsequently developed into an <strong>ACM ICAIF main-track submission</strong>.</p></li><li><p><strong>AlphaAllocator</strong> originated from the <strong>NLPCC Batch</strong> and may later motivate separate financial-ML research.</p></li><li><p>The <strong>NeurIPS Batch</strong> contains several projects developed around the NeurIPS 2026 research cycle, but some are now being extended toward 2027 archival venues.</p></li><li><p>Competition projects such as <strong>Agenthon</strong> and <strong>SimulacraBench</strong> may produce useful research questions or later papers, but their immediate objective is the corresponding benchmark or competition.</p></li></ul><h2>Recruitment status</h2><p><strong>Recruiting</strong> means that new QRS@NTU researchers can currently join the project batch and take ownership of defined research tasks or subproblems.</p><p><strong>Closed</strong> does not necessarily mean that research has ended. It generally means that:</p><ul><li><p>the project team and scope are already established;</p></li><li><p>the work is already close to or past submission;</p></li><li><p>onboarding additional researchers would currently provide limited benefit; or</p></li><li><p>a future follow-on project has not yet been formally opened.</p></li></ul><p>For the current cycle, <strong>new research recruitment is concentrated on the ICAIF / FinRisk Workshop Batch</strong>.</p><h2>Project stages</h2><p>Projects broadly progress through several stages:</p><p><strong>Early research</strong> — the main research question, theorem, method or experiment is still being established.</p><p><strong>Active development</strong> — a promising core result exists, but additional theory, experiments or validation remain.</p><p><strong>Submission preparation</strong> — the scientific package is substantially complete and work is focused on manuscript preparation, review and submission.</p><p><strong>Submitted / under review</strong> — the current paper has been submitted. Further work generally becomes a separate follow-on project rather than modifying the submitted manuscript.</body></html>
</body>
</html>
